Formula/swiftplate.rb

```diff
@@ -0,0 +1,59 @@
+class Swiftplate < Formula
+  desc "Cross-platform Swift framework templates from the command-line"
+  homepage "https://github.com/JohnSundell/SwiftPlate"
+  url "https://github.com/JohnSundell/SwiftPlate/archive/1.2.1.tar.gz",
+    :tag => "1.2.1"
+  sha256 "c4a47411340c88dbacc87d63daf69eed12c80cbc2e1fc8c02cc8d6b2966c04b9"
+  head "https://github.com/JohnSundell/SwiftPlate.git"
+  depends_on :xcode => "8.2"
```

Comment:
@woodruffw We'll probably be needing Swift 3.0.1, so Xcode 8.2 seems accurate. Swift versions are so fragile, best just to require the version you KNOW it works on.